java/compaction/compaction-job-creation/src/main/java/sleeper/compaction/strategy/DelegatingCompactionStrategy.java

```diff
@@ -98,7 +98,8 @@ public List<CompactionJob> createCompactionJobs(List<FileReference> activeFilesW
             if (partition.isLeafPartition()) {
                 compactionJobs.addAll(createJobsForLeafPartition(partition, activeFilesWithJobId, activeFilesWithNoJobId));
             } else {
-                compactionJobs.addAll(createJobsForNonLeafPartition(partition, activeFilesWithNoJobId, partitionIdToPartition));
+                // TODO this will be removed soon, to make checkstyle happy the return value is not used
+                createJobsForNonLeafPartition(partition, activeFilesWithNoJobId, partitionIdToPartition);
```

Comment:
We can just remove this now can't we? There will still be more to do to remove the concept of splitting compactions afterwards.

Reply:
True i didnt realise that the factory method for creating splitting jobs was used in other tests. I've updated this